.github/.codecov.yml

```diff
@@ -15,4 +15,4 @@ coverage:
   status:
     project:
       default:
-        target: 80%
\ No newline at end of file
+        target: 70%
```

Comment:
Any reason to lower the target of the code coverage?

Reply:
What's the plan to bring it back?

Reply:
Nop, no reason. Currently it is less than 50% so 70 sounded reasonable, will add more test to bring it to 70 in next few CRs.  Also, its worth noting that we only copied content from notation-go to this pkg with some additional code (2 files), so tests were already missing.

```
        github.com/notaryproject/notation-plugin-framework-go/plugin    coverage: 26.1% of statements in ./...
ok      github.com/notaryproject/notation-plugin-framework-go/plugin    0.149s  coverage: 26.1% of statements in ./...
github.com/notaryproject/notation-plugin-framework-go/cli/cli.go:30:                    New                                     100.0%
github.com/notaryproject/notation-plugin-framework-go/cli/cli.go:35:                    NewWithLogger                           66.7%
github.com/notaryproject/notation-plugin-framework-go/cli/cli.go:48:                    Execute                                 0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/cli.go:109:                   printVersion                            0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/cli.go:117:                   validateArgs                            0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/cli.go:125:                   unmarshalRequest                        100.0%
github.com/notaryproject/notation-plugin-framework-go/cli/cli.go:133:                   getMetadata                             100.0%
github.com/notaryproject/notation-plugin-framework-go/cli/cli.go:143:                   marshalResponse                         100.0%
github.com/notaryproject/notation-plugin-framework-go/cli/internal/mock/plugin.go:16:   NewPlugin                               100.0%
github.com/notaryproject/notation-plugin-framework-go/cli/internal/mock/plugin.go:20:   DescribeKey                             0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/internal/mock/plugin.go:24:   GenerateSignature                       0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/internal/mock/plugin.go:28:   GenerateEnvelope                        0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/internal/mock/plugin.go:39:   VerifySignature                         0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/internal/mock/plugin.go:64:   GetMetadata                             100.0%
github.com/notaryproject/notation-plugin-framework-go/cli/utils.go:13:                  getValidArgsString                      100.0%
github.com/notaryproject/notation-plugin-framework-go/cli/utils.go:18:                  getValidArgs                            100.0%
github.com/notaryproject/notation-plugin-framework-go/cli/utils.go:40:                  deliverError                            100.0%
github.com/notaryproject/notation-plugin-framework-go/cli/utils.go:47:                  deferStdout                             0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/utils.go:69:                  Debug                                   0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/utils.go:72:                  Debugf                                  0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/utils.go:75:                  Debugln                                 0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/utils.go:78:                  Info                                    0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/utils.go:81:                  Infof                                   0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/utils.go:84:                  Infoln                                  0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/utils.go:87:                  Warn                                    0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/utils.go:90:                  Warnf                                   0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/utils.go:93:                  Warnln                                  0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/utils.go:96:                  Error                                   0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/utils.go:99:                  Errorf                                  0.0%
github.com/notaryproject/notation-plugin-framework-go/cli/utils.go:102:                 Errorln                                 0.0%
github.com/notaryproject/notation-plugin-framework-go/internal/slices/slices.go:4:      Contains                                100.0%
github.com/notaryproject/notation-plugin-framework-go/plugin/describekey.go:10:         Command                                 0.0%
github.com/notaryproject/notation-plugin-framework-go/plugin/errors.go:32:              NewError                                100.0%
github.com/notaryproject/notation-plugin-framework-go/plugin/errors.go:39:              NewGenericError                         100.0%
github.com/notaryproject/notation-plugin-framework-go/plugin/errors.go:43:              NewGenericErrorf                        100.0%
github.com/notaryproject/notation-plugin-framework-go/plugin/errors.go:47:              NewUnsupportedError                     0.0%
github.com/notaryproject/notation-plugin-framework-go/plugin/errors.go:51:              NewValidationError                      100.0%
github.com/notaryproject/notation-plugin-framework-go/plugin/errors.go:55:              NewValidationErrorf                     0.0%
github.com/notaryproject/notation-plugin-framework-go/plugin/errors.go:59:              NewUnsupportedContractVersionError      0.0%
github.com/notaryproject/notation-plugin-framework-go/plugin/errors.go:63:              NewJSONParsingError                     100.0%
github.com/notaryproject/notation-plugin-framework-go/plugin/errors.go:68:              Error                                   75.0%
github.com/notaryproject/notation-plugin-framework-go/plugin/metadata.go:9:             Command                                 0.0%
github.com/notaryproject/notation-plugin-framework-go/plugin/metadata.go:26:            HasCapability                           100.0%
github.com/notaryproject/notation-plugin-framework-go/plugin/sign.go:14:                Command                                 0.0%
github.com/notaryproject/notation-plugin-framework-go/plugin/sign.go:41:                Command                                 0.0%
github.com/notaryproject/notation-plugin-framework-go/plugin/verify.go:16:              Command                                 0.0%
total:                                                                                  (statements)                            42.3%

```

Reply:
We can bump 70 to 80 when we have enough test for 70% coverage.
Also, this is initial setup to get test and other workflows up and running.

Reply:
Created issue https://github.com/notaryproject/notation-plugin-framework-go/issues/9